code-challenges.test.js

```diff
@@ -13,10 +13,41 @@
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
 // a) Create a test with an expect statement using the variable provided.
+
+describe('stFunct1',  () => {
+  it('takes in an array and removes the first item shuffling the remaining content', () => {
+    expect(stFunct1(colors1)).toEqual(expect.arrayContaining(['blue', 'green', 'yellow', 'pink']))
+  })
+})
+
+describe('stFunct2', () => {
```

Comment:
Remember what I've been saying in assessment reviews and intros, you don't need separate tests for each variable.